packages/forms/src/elements/MultiThumbRange.tsx

```diff
@@ -65,6 +66,13 @@ const MultiThumbRange: React.FC<IMultiThumbRangeProps & ThemeProps<DefaultTheme>
   const trackRailRef = useRef<HTMLDivElement>(null);
   const minThumbRef = useRef<HTMLDivElement>(null);
   const maxThumbRef = useRef<HTMLDivElement>(null);
+  const fieldContext = useFieldContext();
+
+  if (fieldContext === undefined) {
+    throw new Error('A MultiThumbRange must be rendered within a form Field component.');
+  }
```

Comment:
I'm not sure this structure should be required. Other elements in the `forms` package mark the `Field` wrapping as optional. 

This would technically be a breaking change as well since it requires a new DOM ordering.

Reply:
I can update the code so that this new behavior is opt-in and not a breaking change. I could also create a v9 task that enforces this change pending what the team thinks.